skin.titan.helix/1080i/IncludesDefaultSkinSettings.xml

```diff
@@ -41,6 +41,7 @@
 		<onload condition="!Skin.HasSetting(HomeMenuDefaults20)">Skin.SetString(OSDPanelOpacity, 60)</onload>
 		<onload condition="!Skin.HasSetting(HomeMenuDefaults20)">Skin.SetString(GuideRows, 9)</onload>
 		<onload condition="!Skin.HasSetting(HomeMenuDefaults20)">Skin.SetBool(GuideShowInfo)</onload>
+		<onload condition="!Skin.HasSetting(HomeMenuDefaults20)">Skin.SetBool(ShowWeatherVideoInfoOSD)</onload>
```

Comment:
remember that by doing this you enable the setting by default. Was that your intention ?